src/app/posts/posts.component.html

```diff
@@ -1,7 +1,7 @@
 <h1>Posts</h1>
 
 <ul>
-  <li *ngFor="let post of posts$">
+  <li *ngFor="let post of posts$ | async">
```

Comment:
Here we are looping through the array that is returned by the observable.  Again, the ```async``` pipe is "unpacking" the observable and we are using that to perform what is needed.